indra/llimage/llimage.cpp

```diff
@@ -657,7 +655,7 @@ void LLImageBase::sanityCheck()
 		|| mComponents > (S8)MAX_IMAGE_COMPONENTS
 		)
 	{
-		LL_ERRS() << "Failed LLImageBase::sanityCheck "
+		LL_WARNS() << "Failed LLImageBase::sanityCheck "
 			   << "width " << mWidth
 			   << "height " << mHeight
 			   << "datasize " << mDataSize
```

Comment:
This function isn't used for anything and was probably left as a debugging tool. `LL_ERRS` seems fine here.